apps/design-land/src/app/radio/radio.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'design-land-radio',
+  templateUrl: './radio.component.html',
+  styleUrls: ['./radio.component.scss']
+})
+export class RadioComponent implements OnInit {
+
+  radioGroup = new FormGroup({
+    race: new FormControl('Terran')
+  });
+
+  constructor() { }
```

Comment:
Remove useless constructor/ngOnInit.

Reply:
Addressed in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d